code/style.css

```diff
@@ -1,7 +1,592 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+/***** 1. GENERAL *****/
 
-.active + .answer {
-  /* some styling here */
+body {
+  box-sizing: border-box;
+  margin: 0;
 }
+
+.background-img {
+  background-image: url("Images/jeans.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100vw;
+  height: 100vh;
+}
+
+h2,
+h3,
+p {
+  font-family: 'Varela', sans-serif;
+  margin: 0;
+}
+
+/***** 2. HEADER AND NAV *****/
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding-top: 15px;
+  padding-right: 10px;
+  padding-left: 10px;
+}
+
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  width: 30px;
+  height: 20px;
+}
+
+.hamburger div {
+  width: 30px;
+  height: 4px;
+  background: white;
+  border-radius: 10px;
+}
+
+.content {
+  display: none;
+}
+
+.content a {
+  font-family: 'Varela', sans-serif;
+  font-size: 10px;
+  color: #fff;
+  text-transform: uppercase;
+  text-decoration: none;
+  text-shadow: 2px 2px 3px rgb(98, 93, 110);
+}
+
+.content a:hover {
+  transform: translateY(2px); 
+}
+
+.nav-sign-in {
+  display: inline;
+  align-self: flex-start;
+}
+
+.nav-sign-in a {
+  font-family: 'Varela', sans-serif;
+  font-size: 10px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: rgb(45, 44, 51);
+}
+
+/***** 3. LOGO *****/
+
+.logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 150px;
+  height: 150px;
+  border: solid 3px black;
+  border-radius: 50%;
+  background: #ffffff;
+  margin: auto;
+  margin-top: 80px;
+}
+
+.name {
+  font-family: 'Lobster', cursive;
+  font-size: 80px;
+  margin: 0;
+}
+
+/***** 4. INFO *****/
+
+.info {
+  padding: 10px;
+  margin-top: 50px;
+  text-align: center;
+  backdrop-filter: blur(10px);
+  -webkit-backdrop-filter: blur(10px);
+}
+
+.info h2 {
+  font-size: 25px;
+  color: #fff;
+  width: 90%;
+  margin: auto;
+}
+
+.info p {
+  font-size: 14px;
+  color: #fff;
+  width: 90%;
+  margin: auto;
+}
+
+/***** 5. SIGN UP BUTTON *****/
+
+.signup-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 100vw;
+  height: 70px;
+  background: #fff;
+  position: fixed;
+  bottom: 0;
+  right: 0;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+  box-shadow: 0 -2px 5px #3b383f;
+}
+
+.sign-up {
+  display: flex;
+
+  width: 80%;
+  height: 50px;
+  margin: auto;
+  border: solid 1px black;
+  border-radius: 30px;
+  background: rgb(91, 100, 107);
+
+  background-image: url("Images/fabric.jpg");
```

Comment:
All folders should have lowercase names - once again, details in [Week 2 Master Document](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)